app/src/main/java/com/infomaniak/drive/ui/MainViewModel.kt

```diff
@@ -81,8 +87,36 @@ class MainViewModel(appContext: Application) : AndroidViewModel(appContext) {
     private var getFileDetailsJob = Job()
     private var syncOfflineFilesJob = Job()
 
+    lateinit var uploadFilesHelper: UploadFilesHelper
```

Comment:
We probably want to avoid using a lateinit for this, as it would lead to crash if it's used without a call to initFirst (which will certainly arrive)